src/kibana/plugins/dashboard/styles/main.less

```diff
@@ -72,8 +72,9 @@ dashboard-grid {
 
         .panel-title {
           font-size: inherit;
+          display: inline-block;
           .ellipsis();
-          .flex(1 1 0);
+          .flex(1 1 0%);
```

Comment:
This flex property as far as I can tell is invalid, and I don't understand why it works. :)
Please change line 77 to be:
```
.flex(1 1 auto);
```